sunpy/map/sources/sdo.py

```diff
@@ -117,4 +119,32 @@ def observatory(self):
     @classmethod
     def is_datasource_for(cls, data, header, **kwargs):
         """Determines if header corresponds to an HMI image"""
-        return str(header.get('instrume', '')).startswith('HMI')
+        return (str(header.get('TELESCOP', '')).endswith('HMI') and
+                not HMISynopticMap.is_datasource_for(data, header))
+
+
+class HMISynopticMap(HMIMap):
+
+    def __init__(self, data, header, **kwargs):
+        super().__init__(data, header, **kwargs)
+
+        if self.meta['cunit1'] == 'Degree':
+            self.meta['cunit1'] = 'degree'
+
+        if self.meta['cunit2'] == 'Sine Latitude':
+            self.meta['cunit2'] = 'degree'
+
+            # Since, this map uses the cylindrical equal-area (CEA) projection,
+            # the spacing should be modified to 180/pi times the original value
+            # Reference: Section 5.5, Thompson 2006
+            self.meta['cdelt2'] = 180 / np.pi * self.meta['cdelt2']
+            self.meta['cdelt1'] *= np.abs(self.meta['cdelt1'])
```

Comment:
An extra * has snuck in here
```suggestion
            self.meta['cdelt1'] = np.abs(self.meta['cdelt1'])
```